src/main/scala/notifications/Email.scala

```diff
@@ -24,8 +27,8 @@ class Email extends Actor {
     properties.put("mail.smtp.port", port)
     properties.put("mail.smtp.auth", "true")
     properties.put("mail.smtp.starttls.enable", "true")
-    properties.put("mail.smtp.connectiontimeout", "10000")
-    properties.put("mail.smtp.timeout", "10000")
+    properties.put("mail.smtp.connectiontimeout", timeout)
+    properties.put("mail.smtp.timeout", timeout)
```

Comment:
all these properties setting should be in a separate method. In their current form they should all get kicked out of `createMessage` since currently you are just passing new objects with exactly the same properties to `Session.getInstance` (there is nothing dynamic about setting these properies, all fields don't change)

Reply:
ok, i decided to change both properties and authenticator to class fields